Client/src/Validation/validation.js

```diff
@@ -128,10 +128,51 @@ const maintenanceWindowValidation = joi.object({
 	monitors: joi.array().min(1),
 });
 
+const advancedSettingsValidation = joi.object({
+	apiBaseUrl: joi.string().uri({ allowRelative: true }).trim().messages({
+		"string.empty": "API base url is required.",
+		"string.uri": "The URL you provided is not valid.",
+	}),
+	logLevel: joi.string().valid("debug", "none", "error", "warn").allow(""),
+	systemEmailHost: joi.string().allow(""),
+	systemEmailPort: joi.string().allow(""),
+	systemEmailAddress: joi.string().allow(""),
+	systemEmailPassword: joi.string().allow(""),
+	jwtTTLNum: joi.number().messages({
+		"number.base": "JWT TTL is required.",
+	}),
+	jwtTTLUnits: joi
+		.string()
+		.trim()
+		.custom((value, helpers) => {
+			if (!["days", "hours"].includes(value)) {
+				return helpers.message("JWT TTL unit is required.");
+			}
+			return value;
+		}),
+	dbType: joi.string().trim().messages({
+		"string.empty": "DB type is required.",
+	}),
+	redisHost: joi.string().trim().messages({
+		"string.empty": "Redis host is required.",
+	}),
+	redisPort: joi
+		.string()
```

Comment:
This should be number as well

Reply:
You seems not having the latest code pulled to your local? the email port and redis port are all optional, to be consistent with BE validation, so if you left empty, there is no error message; (see below screenshot);  if you try to enter anything, only a number can be entered;   I change the label for system email port, see below
![Screenshot from 2024-10-25 10-01-24](https://github.com/user-attachments/assets/ee741849-93cb-456a-b010-47082168e003)

Reply:
Resolved in : https://github.com/bluewave-labs/bluewave-uptime/pull/1020/commits/2427bdb78804732071a288e4d69c54f43bae804f